packages/tictactoe/src/redux/game/__tests__/player-b.test.ts

```diff
@@ -256,4 +256,23 @@ describe("player B's app", () => {
       });
     });
   });
+  describe("when in PlayAgain", () => {
+    const gameState = state.playAgain({
+      ...bProps,
+      ...draw,
+      result: Result.Tie,
+    });
+
+    describe("if the player decides to continue", () => {
+      const action = actions.playAgain();
+      const updatedState = gameReducer({ messageState, gameState }, action);
+
+      itIncreasesTurnNumBy(1, { gameState, messageState }, updatedState);
+      itSends(scenarios.swapRoles.resting2, updatedState);
+      itTransitionsTo(state.StateName.WaitToPlayAgain, updatedState);
+      // TODO check that whomever did not play last sends resting
```

Comment:
@geoknee Should this TODO get resolved in this PR?

Reply:
Good point. I resolved it, and commit message has some ideas of a better approach in the future. (This PR was about fixing the reducer, rather than the tests, so I won't implement those ideas here).